### PR TITLE
Makefile: don't pre-compress manpages

### PR DESCRIPTION
**Pull request by eli-schwartz**
The man program supports a variety of compressed manpage formats, as well as uncompressed manpages. It is standard practice for software to decline to make assumptions about what individual redistributors want to do.

Various distros have different policies about how to handle this:

- Arch Linux compresses manpages that haven't yet been compressed, conditional on a tuneable site option "zipman"

- Alpine compresses manpages that haven't yet been compressed

- debian automatically compresses manpages that haven't yet been compressed with dh_compress

- fedora discards all compression you do and recompresses with brp-compress

- Gentoo issues a QA warning if it is already compressed, and if uncompressed will allow you to configure via:
  - PORTAGE_COMPRESS, which compression method to use (defaulting to bzip2, not gzip)
  - PORTAGE_COMPRESS_EXCLUDE_SUFFIXES, whether to compress at all

- voidlinux finds all manpages installed with compression, and uncompresses them

In short, no one actually wants packages to install compressed manpages directly. Furthermore, the gzip program isn't reproducible by default, but distros that offer optional or mandatory (re)compression of man pages pass the requisite options to make it reproducible.

### CODE REVIEW

**Comment by eli-schwartz** [comment]
Thanks. By the way I think you accidentally pushed only the tag, but not the commit incrementing the version. :)

**Comment by graysky2** [comment]
thanks
